colossalai/utils/profiler/_base_profiler.py

```diff
@@ -0,0 +1,28 @@
+from torch.utils.tensorboard import SummaryWriter
+
+
+class BaseProfiler(object):

```

Comment:
make it an abstract class with abc

Reply:
derive from ABC if it's an abstract class